packages/react-core/src/components/Tabs/TabTitleIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Tabs/tabs';
+
+export interface TabTitleIconProps extends React.HTMLProps<HTMLSpanElement> {
+  /** Icon to be rendered inside the tab button title. */
+  children?: React.ReactNode;
```

Comment:
i'd make this required

Reply:
We do not generally make children for similar components required. Is it because the title is required?

Reply:
without children this component does nothing right? so that's why i think it's not optional. when else should something be required? :)